bioblend/galaxy/objects/wrappers.py

```diff
@@ -193,7 +193,7 @@ class Workflow(Wrapper):
     results from an input dataset.
     """
     BASE_ATTRS = Wrapper.BASE_ATTRS + (
-        'deleted', 'inputs', 'published', 'steps', 'tags'
+        'deleted', 'inputs', 'published', 'steps', 'tags', 'owner', 'number_of_steps', 'show_in_tool_panel', 'latest_workflow_uuid'
```

Comment:
I think `number_of_steps` and `show_in_tool_panel` are not presently available in `/api/workflows/<workflow_id>` , but only in `/api/workflows/`, which means that they can only be added to `BASE_ATTRS` of the `WorkflowPreview` class.

Reply:
Fixed